.github/workflows/pull_request.yml

```diff
@@ -27,3 +27,11 @@ jobs:
 
       - name: Build all configurations
         run: ./gradlew assemble
+
+      - name: Upload APK as Artifacts
+        uses: actions/upload-artifact@v2.2.0
+        with:
+          name: kiwix-android
+          path: |
+            app/**/*.apk
```

Comment:
I am not sure what you tried but I think these 2 lines are replaceable with `**/**.apk`

Reply:
or `**/*.apk` it is always tough to say what regex syntax somebody is using

Reply:
Yep, I used it but it was a wrong way I remember it, I used it like:
`path: **/*.apk`

It then didn't even trigger the CI, so I removed it.